eq-author-api/middleware/identification/verifyJwtToken.js

```diff
@@ -0,0 +1,49 @@
+const fetch = require("node-fetch");
+const jwt = require("jsonwebtoken");
+
+const {
+  VERIFY_PUBLIC_KEYS_URL,
+  VERIFY_VALIDITY_OPTIONS,
+} = require("../../constants/googleServiceKeys");
+
+let googlePublicKeys;
+
+const fetchUpdatedKeys = async () => {
+  try {
+    const response = await fetch(VERIFY_PUBLIC_KEYS_URL);
+
+    googlePublicKeys = await response.json();
+  } catch (error) {
```

Comment:
Can we do something with this error instead of suppress it?

Reply:
If we can't fetch the keys I don't think there's any way to reasonably recover, so we could just throw the error here.

Reply:
I wasn't sure how to respond to this error. It will occur if the Google public service keys aren't available, which is unlikely. How about the following, which should keep the site running if keys are unavailable:

1. Log error
2. Set the googlePublickKeys variable to null. 
3. When it comes to verifying, if googlePublicKeys is null then just do a decode.
4. Next request will attempt fetch of googlePublicKeys

Reply:
I agree we should log the error, but I don't think we should allow unverified tokens to call our API.

Reply:
Could we just crash?

Reply:
Probably, I think if Google is down we have bigger problems anyway.